contracts/token/BiconomyTokenPaymaster.sol

```diff
@@ -181,7 +158,7 @@ contract BiconomyTokenPaymaster is BasePaymaster, ReentrancyGuard, TokenPaymaste
         emit NewFeeTokenSupported(_token, msg.sender);
     }
 
-    function setUnaccountedEPGasOverhead(uint256 value) external onlyOwner {
+    function setUnaccountedEPGasOverhead(uint256 value) external payable onlyOwner {
```

Comment:
All the methods that are not likely to be called (for example onlyOwner ones) will save gas if they are payable, because this removes the verification if there's value provided with the call